bin/docker/run

```diff
@@ -35,6 +39,9 @@ beaver_vars="DIST"
 if [ $pass_credentials_bintray ]; then
     beaver_vars="$beaver_vars BINTRAY_USER BINTRAY_KEY"
 fi
+if [ $pass_credentials_github ]; then
+    beaver_vars="$beaver_vars GITHUB_OAUTH_TOKEN"
+fi
```

Comment:
Sort of similar with this. I prefer to have more specialized commands than adding so many options to run.

For the specific use case of pushing automatically converted D2 tags, I would definitely create a beaver command. My idea was to start moving some common D building facilities to beaver (like the DMD -> DC script, etc.). This is why I also added subcommands. I see a `beaver dlang push-d2-tags` for example.